network_windows.go

```diff
@@ -30,6 +30,10 @@ func (n *network) startResolver() {
 		options := n.Info().DriverOptions()
 		hnsid := options[windows.HNSID]
 
+		if hnsid == "" {
+			return
+		}
+
```

Comment:
This is expected for none network. The only way to know that a docker network is not backed by an HNS network is to query the driver options. To me this looks like the best place to have that check.